README.md

```diff
@@ -11,4 +10,140 @@ UAVCAN is an open lightweight protocol designed for reliable communication in ae
 robust vehicle bus networks.
 
 * [**UAVCAN website**](http://uavcan.org)
-* [**UAVCAN mailing list**](https://groups.google.com/forum/#!forum/uavcan)
+* [**UAVCAN forum**](https://forum.uavcan.org)
+
+## Identifier ranges
+
+### Subjects
+
+For message subjects, the following range mapping is adopted (limits inclusive).
+Unused ranges are reserved for future expansion of adjacent ranges.
+
+From    | To        | Purpose
+--------|-----------|------------------------------------------------
+0       | 32767     | Application-specific unregulated identifiers (freely chosen by the integrator)
+57344   | 59391     | Vendor-specific fixed identifiers (stored in a public repository)
+62804   | 65535     | Standard fixed identifiers
+
+### Services
+
+For services, the following range mapping is adopted (limits inclusive).
+Unused ranges are reserved for future expansion of adjacent ranges.
+
+From    | To        | Purpose
+--------|-----------|------------------------------------------------
+0       | 127       | Application-specific unregulated identifiers (freely chosen by the integrator)
+256     | 319       | Vendor-specific fixed identifiers (stored in a public repository)
+384     | 511       | Standard fixed identifiers
+
+## Standard static identifier allocation
+
+### Subjects
+
+Ordered by priority from high to low.
+
+Namespace                   | Lower bound (inclusive)
+----------------------------|-------------------------
+`uavcan.time`               | 62804
+`uavcan.node`               | 62805
+`uavcan.pnp`                | 62810/65533
```

Comment:
I'm not a fan of the PNP name. There have been so many security-hole-riddled things with the name PNP that it's sort of tainted. Furthermore, I'm interested in the assignment of dynamic node IDs as a one-time operation which doesn't really fit into the concept of PNP.

How about `uavcan.config` instead? This is ultimately a configuration step and there may be other standard configuration functions that emerge in the future.

Reply:
Eh, our node ID allocation protocol does not concern itself with security against malicious nodes either, that would be outside of the scope of the protocol. Just wash your ~hands~ nodes before connecting them to the bus.

Config is bad, sounds like something about configuration parameters. Let's look for better names please. I personally like (despite bad PR) PNP because it's to the point and you understand what it's about when you look at it, but I don't insist on that.

> I'm interested in the assignment of dynamic node IDs as a one-time operation which doesn't really fit into the concept of PNP.

It is, by the very definition of the protocol, a one-time operation, and it is guaranteed even by the current spec v0 (quote from uavcan.org):

> During dynamic allocation, the allocatee communicates its unique ID to the allocator (or allocators), which then use it to produce an appropriate allocation response. Unique ID values are kept by allocators in allocation tables -- data structures that contain mappings between unique ID and corresponding node ID values. **Allocation tables are write-only data structures that can only grow. Once a new allocatee has requested a node ID, its unique ID will be recorded into the allocation table, and all subsequent allocation requests from the same allocatee will be served with the same node ID value.**

Reply:
okay. pnp is fine.